clients/src/main/java/org/apache/kafka/common/protocol/Errors.java

```diff
@@ -20,26 +20,26 @@
 import java.util.Map;
 
 import org.apache.kafka.common.errors.*;
```

Comment:
This is causing a merge conflict, and it is now invalid to use star imports. Should be a minor fix, but there's a sizable list of imports that this needs to be replaced with.